packages/core/src/Stores/client-store.js

```diff
@@ -2689,4 +2697,95 @@ export default class ClientStore extends BaseStore {
             this.setShouldShowEffortlessLoginModal(false);
         }
     }
+
+    addSubscription(name, key, subscription, subscription_id) {
+        if (!this.subscriptions[name]) {
+            this.subscriptions[name] = {};
+        }
+        if (!this.subscriptions[name][key]) {
+            this.subscriptions[name][key] = { sub: undefined, id: undefined };
+        }
```

Comment:
this.subscriptions[name] = this.subscriptions[name] ?? {};
    this.subscriptions[name][key] = this.subscriptions[name][key] ?? { sub: undefined, id: undefined };